tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -3,10 +3,18 @@ summary: Ensure upgrades from snapd 2.15 work
 systems: [ubuntu-16.04-64]
 
 prepare: |
-    dpkg --purge snapd
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB/pkgdb.sh"
+    distro_purge_package snapd
 
 restore: |
-    dpkg --purge ubuntu-core-launcher snap-confine
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB/pkgdb.sh"
+    if [ -e old_install_done ]; then
+        echo "Ensure core is gone and we have ubuntu-core instead"
+        distro_purge_package snapd
```

Comment:
```suggestion
        distro_purge_package snapd
        rm old_install_done
```

Reply:
it is not needed "rm -f old_install_done" due to the test directory is restored automatically.